sfdx-source/apex-common/main/classes/fflib_SObjects.cls

```diff
@@ -257,7 +257,7 @@ public virtual class fflib_SObjects
 		{
 			for (SObjectField field : fields)
 			{
-				if (String.isNotBlank((String) record.get(field))) continue;
+				if (String.isNotBlank(String.valueOf(record.get(field)))) continue;
```

Comment:
@ImJohnMDaniel  You're a stickler on using explicit blocks for if statements.  I think that would be a good measure here too, especially since I had to look at it several time to see why this method wasn't doing the opposite of its name.  😇